### DISCUSSION TITLE
Saved location for Dropped videos from external HD

### DISCUSSION THREAD

**Discussion by maddust** [opendatacam/opendatacam#531] [Q&A] [answered]
Sorry for the dumb question.. but i will like to know where the dropped videos are saved when dropped directly to the UI.from an external HD 

im currently counting very large files (24hr videos) and I'm concerned about my disk space.  currently, im using the Docker installation on a desktop. 

Also, i noticed also that the FPS caps out depending on the encoding and file extensions of the videos .. For example, the test.mp4 file that comes by default runs on my pc at 255fps .  and if i drop an .AVI (h264) 1080p... the fps doesn't go higher than 32 fps ..  no matter what weights i use tiny or regular yolov4.

any recommendations about ideal file encoding and extension will be appreciate it

**Comment by vsaw** [comment, marked as answer]
Videos uploaded via Drag and Drop get stored temporarily in a folder inside the docker container which is not shared with the host.

This can currently not be changed, but what you can do is share the video with the container via the docker-compose.yaml and set the config to use the file. This will run the file directly from the external drive. 

As for the FPS cap 🤷‍♂️

There is no built in cap and I can run with Yolov4 Tiny at 60fos on a Xavier NX for example without any problems. Can you look at your system performance and stats to figure this out?

**Reply by maddust** [reply]
thanks for the answer , so the temporarily files get deleted once i restart ODC? 

for the FPS issue , i will investigate more... i seems very strange that both Yolo versions run at the same FPS... maybe i have something set wrong . 

im running a RTX 2070 on a Rayzen 5 ,32GB Ram , SSD.... should perform "decent" .

**Reply by vsaw** [reply]
> thanks for the answer , so the temporarily files get deleted once i restart ODC?

That is correct.
